addons/main/script_macros.hpp

```diff
@@ -269,18 +273,72 @@
 
 #define ALL_BODY_PARTS_PRIORITY ["body", "head", "leftarm", "rightarm", "leftleg", "rightleg"]
 
+#define DEFAULT_PACO2 40
+#define DEFAULT_PAO2 90
+#define DEFAULT_O2SAT 0.96
+#define DEFAULT_HCO3 24
+#define DEFAULT_PH 7.4
+#define DEFAULT_ETCO2 37
+#define DEFAULT_BLOOD_GAS [DEFAULT_PACO2, DEFAULT_PAO2, DEFAULT_O2SAT, DEFAULT_HCO3, DEFAULT_PH, DEFAULT_ETCO2]
+
+#define DEFAULT_ANEROBIC_EXCHANGE 0.8
+#define DEFAULT_TEMPERATURE 37
+#define DEFAULT_STROKE_VOLUME 0.001583333323
+#define DEFAULT_FIO2 0.21
+
+#define DEFAULT_ECB 2700
+#define DEFAULT_ECP 3300
+#define DEFAULT_SRBC 500
+#define DEFAULT_ISP 10000
+#define DEFAULT_BODY_FLUID [2700, 3300, 500, 10000, 6000]
+#define LITERS_TO_ML 1000
+#define ML_TO_LITERS 1000
+
 // Airway
-#define VAR_SPO2                       QEGVAR(breathing,airwayStatus)
-#define GET_SPO2(unit)                 (unit getVariable [VAR_SPO2, 100])
+#define OXYGEN_PERCENTAGE_CRITICAL 85
```

Comment:
first of all, why do we define stuff that was always configurable
second, this macro is not used in whole PR